zwave_js_server/exceptions.py

```diff
@@ -86,7 +86,7 @@ def __init__(
         self, message_id: str, error_code: str, msg: str | None = None
     ) -> None:
         """Initialize a failed command error."""
-        super().__init__(msg or f"Command failed: {error_code}")
+        super().__init__(f"{error_code}: {msg}" or f"Command failed: {error_code}")
```

Comment:
We can't write the condition like that now since the first part is always true.

Reply:
good point